beta-src/src/interfaces/PhaseScroll.ts

```diff
@@ -1,5 +1,16 @@
 import { ScrollButtonState } from "../enums/UIState";
 
+interface gameState {
+  seasons: string[];
```

Comment:
the season are most likely an enum or already are. if not, they should so this would be something like

Seasons[]